src/w3c/headers.js

```diff
@@ -265,6 +265,9 @@ export function run(conf) {
       pub("warn", msg);
     }
   }
+  if (validLevel(conf.level)) {
+    conf.shortName = `${conf.shortName}-${conf.level}`;
```

Comment:
This should happen in the "w3c/levels" plugin - so then it's all in one place (and then we don't need the util function anymore).